draft-wang-ppm-differential-privacy.md

```diff
@@ -217,19 +217,18 @@ in DAP. It does so in the following stages:
 
 1. {{mechanisms}} defines an interface for DP mechanisms that are compatible
    with VDAFs and, for all VDAFs specified in {{!VDAF}} [TODO: And other
-   drafts, once they appear], specifies concrete Client- and Aggregator-DP
-   mechanisms.
+   drafts, once they appear], specifies concrete mechanisms.
 
-1. {{policy}} specifies the integration of DP mechanisms from the previous
-   section into DAP. In particular, it describes how a DP policy is implemented
-   by the Client, Aggregator, and Collector alongside the execution of the
-   VDAF.
+1. {{dp-in-dap}} specifies the integration of DP mechanisms from the previous
```

Comment:
I think there is still a concept of policy missing, a policy is implemented by DpMechanism but it may have concerns that’s not covered by DpMechanism, for eg. A policy of DP amplification depends on batch size, if it’s larger than min batch size, no noise needs to be added. I think this decision is the role of a policy type, which should be _used_ by DAP aggregators

Reply:
This falls into the bucket of changes for DAP. I suggested to Junye that we avoid mentioning protocl changes for now and just carve out a section in the draft for them. I do plan to mention this optmization in the PPM meeting however.

Reply:
Also, as discussed internally, we're going to merge this then rename "primitive" to "mechanism" and "mechanism" to "policy". Please plan on doing a full pass of the draft when you wake up and either file a PR to fix it up or drop comments somewhere :)

Reply:
Done.